src/parsers/insertmode.ts

```diff
@@ -6,6 +6,8 @@ export function parser(keys){
     if (! hasModifiers(keys[0])) {
         if (keys[0].key === 'Escape') {
             return {keys: [], ex_str: 'unfocus'}
+        } else if (keys[0].key === 'Tab') {
+            return {keys: [], ex_str: 'focusinput -n'}
```

Comment:
This functionality is invoked whenever in 'Insert Mode' (virtually whenever an input field is selected) and not just after 'focusinput -l'. Was this your intention? Because I maintain a hint mode selecting only input fields is more suited for that and this is a change in core behavior that I feel deserves a setting.